docs/quick-starts/geolift-quickstart.md

```diff
@@ -0,0 +1,35 @@
+---
+slug: /geolift-quickstart
+sidebar_position: 8
+---
+# Creating your first Geolift test
+
+This guide will walk you through creating your first Eppo Geolift quasi-experiment. We will create a new quasi-experiment, specify the business question, specify test parameters, and submit the test to the power analysis and simulation pipeline.
+
+1. **Create the aggregation Entity**: First, create an Entity in Eppo that defines the unit of randomization you want to use for the quasi-experiment. This is often geographic in nature. It should match what’s available in your data warehouse in the following step. More information on entities here. For example, if your test is going to split on countries, you might create a "Country" entity.
+2. **Create a Fact SQL for the events to be measured**: If you want to test the intervention’s effect on revenue by Country, you’ll want to create a Fact SQL that brings net revenue into Eppo along with the Country that the revenue took place in. More information on Facts here.
+3. **Create a Metric for the event to be measured**: Next, create a metric that aggregates the revenue by country created in the previous step into the metric that you’d like to examine in this test. When creating a new metric, select the Entity you just created, then the Fact and the Aggregation type desired. More information on Metrics here. Eppo Geolift currently supports SUM and COUNT metrics with no timeframes specified.
+
+## Test objective and data
+
+1. **Create a Quasi-experiment**: Now, you can specify the parameters for the Geolift test.
```

Comment:
Describe where to create a quasi experiment (In the analysis tab, click the create button and select quasi-experiment)